packages/plugin-lightningcss/src/minimizer.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * modified from https://github.com/fz6m/lightningcss-loader
+ * MIT License https://github.com/fz6m/lightningcss-loader/blob/main/LICENSE
+ * Author @fz6m
+ */
+import { transform as _transform } from 'lightningcss';
+import { Buffer } from 'node:buffer';
+import type { Compilation, Compiler } from 'webpack';
+import type { LightningCssMinifyPluginOptions } from './types';
+
+const PLUGIN_NAME = 'lightningcss-minify-plugin';
+const CSS_FILE_REG = /\.css(?:\?.*)?$/i;
```

Comment:
can we use `CSS_REGEX` from `@rsbuild/shared`. / when will the `(?:\?.*)` be matched.